### DISCUSSION TITLE
Welcome to nepal-compliance Discussions!

### DISCUSSION THREAD

**Discussion by njsubedi** [yarsa/nepal-compliance#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by surajshres** [comment]
Hey guys!
Any timeline on when CBMS integration be completed?

**Reply by sparshrestha** [reply]
Wow! Thank you for following our work. @S-mkc can share more details on this.

**Reply by S-mkc** [reply]
Hyy @surajshres, This is our next step. We are currently undergoing validation. Once that is completed, we will proceed with the CBMS integration. We look forward valuable feedbacks. Thank You

**Reply by surajshres** [reply]
Thanks. I look forward to it.

**Comment by alok7318** [comment]
Hello Everyone, 
Can anyone tell m how can I get IRD registration for ERPNEXT ?
